pom.xml

```diff
@@ -100,6 +104,7 @@
     <prometheus.version>0.8.0</prometheus.version>
     <http.version>4.4.1</http.version>
     <spark.version>2.4.4</spark.version>
+    <spark2.version>2.4.4</spark2.version>
```

Comment:
I would suggest just keeping `spark.version` here. Override the `spark.version` respectively in `hudi-spark2` and `hudi-spark3` modules.

Reply:
Does Maven support overriding ```spark.version``` only in sub modules?

Reply:
If any dependency or property is configured in both parent and child POMs with different values then the child POM value will take the priority.